examples/body-to-query-params-server/app.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+app.use(bodyParser.json())
+const axios = require('axios');
+const { json } = require('stream/consumers');
+
+const port = 3000
+
+app.post('/', (req, res) => {
+    const headerOutboundUrl = req.get('Outbound-Url');
+    console.log('Outbount Url: ' + process.env.OUTBOUND_URL);
+    const outboundUrl = headerOutboundUrl + "?";
```

Comment:
Please use the URL api from node instead of constructing this stuff by hand. It will handle quite a bit of the edge cases that you may run into doing this
https://nodejs.org/api/url.html